hack/auto-cp.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env sh
+
+set -eu
+# this script prints out a list a commits that are on master that you should probably cherry-pick to the release branch
+
+br=$1;# branch
+commitPrefix=$2;# examples: fix, chore(deps), build, ci
+commitGrepPattern="^${commitPrefix}(*.*)*:.*(#"
+
+# find the branch point
+base=$(git merge-base $br master)
+
+# extract the PRs from stdin
+prNo() {
+  set -eu
+  cat | sed 's|.*(\(#[0-9]*\))|\1|'
+}
+
+# list the PRs on each branch
+prs() {
+  set -eu
+  git log --format=%s --grep ${commitGrepPattern} $1...$2 | prNo | sort > /tmp/$2
+}
+
+prs $base $br
+prs $base master
+
+# find PRs added to master
+diff /tmp/$br /tmp/master | grep '^> ' | cut -c 3- > /tmp/prs
+
+# print all the commits that need cherry-picking
+git log --oneline --grep ${commitGrepPattern} $base...master | while read -r m; do
+  commit=`grep -q "$(echo $m | prNo)" /tmp/prs && echo ${m:0:9}`
+  set +e
+  echo cherry-picking: $commit
+  if ! git cp $commit; then
+    echo failed to cherry-pick $commit
+    git cp --abort
+  fi
```

Comment:
this is the main difference, right? I would think that it probably makes more sense to add a flag to `what-to-cherry-pick.sh` instead of copy+pasting a whole script and needing to keep both in sync.

the default behavior would basically be a "dry run" then.

or alternatively you could pipe the results of `what-to-cherry-pick.sh` into this script